app/src/main/java/com/khinthirisoe/bakingapp/data/db/IngredientsContract.kt

```diff
@@ -0,0 +1,55 @@
+package com.khinthirisoe.bakingapp.data.db
+
+import android.content.ContentResolver
+import android.net.Uri
+import android.provider.BaseColumns
+
+object IngredientsContract {
+
+    const val CONTENT_AUTHORITY = "com.khinthirisoe.bakingapp"
+    val BASE_CONTENT_URI: Uri = Uri.parse("content://$CONTENT_AUTHORITY")
+
+    const val TYPE_TEXT = "TEXT"
+    const val PRIMARY_KEY = " PRIMARY KEY"
+    const val NOT_NULL = " NOT NULL"
+    const val UNIQUE = " UNIQUE"
+    const val TYPE_PRIMARY_KEY_TEXT_UNIQUE = TYPE_TEXT + PRIMARY_KEY + NOT_NULL + UNIQUE
+    const val TYPE_TEXT_NOT_NULL = TYPE_TEXT + NOT_NULL
```

Comment:
All of these constants should be in DbConstatns because they will be used in other table contracts

Reply:
alright

Reply:
fixed it